clients/drcachesim/common/trace_entry.h

```diff
@@ -191,6 +191,32 @@ typedef enum {
      */
     TRACE_MARKER_TYPE_CPU_ID,
 
+    /**
+     * The marker value contains the function id defined by user in droption
+     */
+    TRACE_MARKER_TYPE_FUNC_ID,
+
+    /**
+     * The marker value contains the return address of the function call,
+     * whose id is specified by the closest previous FUNC_ID marker entry
```

Comment:
Better to get a link by using the full name and `#`: `#TRACE_MARKER_TYPE_FUNC_ID`.

Missing `.` again (let's keep these user-facing doxygen comments professional-looking)

Reply:
Done